src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -98,14 +130,14 @@ def get_ansys_version() -> str:
 
     Raises
     ------
-    RuntimeError
+    AnsysVersionNotFound
         If an Ansys version cannot be found.
     """
     for v in FluentVersion:
         if "AWP_ROOT" + "".join(str(v).split("."))[:-1] in os.environ:
             return str(v)
 
-    raise RuntimeError("An Ansys version cannot be found.")
+    raise AnsysVersionNotFound("An Ansys version cannot be found.")
```

Comment:
The error message should be informative, so that the end-user has a chance to resolve it. The message shouldmention something about AWP_ROOT env vars.

Reply:
Yes, I agree.

Reply:
Done. Thank you.